src/lodf_calculations.jl

```diff
@@ -413,13 +442,25 @@ function LODF(
     validate_linear_solver(linear_solver)
     ax_ref = make_ax_ref(A.axes[1])
     if reduce_radial_branches
-        radial_branches = RadialBranches(A)
+        if !isempty(BA.radial_branches) && !isempty(ABA.radial_branches)
```

Comment:
we need to check that the radial branches object is in fact the same

Reply:
done